src/text/Text.story.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { color } from '../_utils/branding'
+import { Text, TextDisplayType, TextTagType } from './index'
+
+<Meta title="Text" />
```

Comment:
Does it means
Text => Default on the SideBar?

Reply:
Yes